src/main/java/com/dougnoel/sentinel/system/DownloadManager.java

```diff
@@ -384,4 +389,23 @@ public static void setDownloadDirectory(String downloadDirectory) {
     public static void clearDownloadDirectory() throws IOException {
         FileUtils.cleanDirectory(new File(downloadDirectory));
     }
+
+    /**
+     * Sets the path of the most recently-downloaded file to the current download directory + the given filename.
+     * @param filename String name of the file downloaded.
+     */
+    private static void setMostRecentDownloadPath(String filename){
```

Comment:
Shame we aren't in C# and can't use {get; set; }

Reply:
agreed